src/server/pachyderm_test.go

```diff
@@ -10676,14 +10701,23 @@ func TestDatumSetCache(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	go func() {
-		ticker := time.NewTicker(time.Minute)
+		ticker := time.NewTimer(50 * time.Second)
 		defer ticker.Stop()
 		for {
 			select {
 			case <-ctx.Done():
 				return
 			case <-ticker.C:
+				// ticker is too smart and will adjust to always go at 1 minute, even if etcd is slow.
```

Comment:
Why not time.After then?  We create a new ticker every iteration, seems like it would be easier to use time.After which will only tick once.

Comment is also slightly outdated; Minute changed to 50 seconds here.

Reply:
changed. I didn't know time.After was an option.